lib/srv/db/mysql/protocol/fuzz_test.go

```diff
@@ -0,0 +1,44 @@
+//go:build go1.18
```

Comment:
I think we're not on 1.18 yet, and v9 and v8 are still (and will be) built by Go 1.17 so backports will probably require regular tests?